code/modules/mob/living/simple_mob/subtypes/mechanical/hivebot/hivebot.dm

```diff
@@ -1,6 +1,7 @@
 // Hivebots are tuned towards how many default lasers are needed to kill them.
 // As such, if laser damage is ever changed, you should change this define.
-#define LASERS_TO_KILL * 40
+//chompedit: Reduce health of hivebots by reducing 40 to 15
+#define LASERS_TO_KILL * 15
```

Comment:
Put the chompEDIT comment after, not above, for single line edits.

Reply:
That is to say on the same line I believe.